### PR TITLE
Init Chroma and LLamaIndex Multi-Modal Demo

### PR DESCRIPTION
**Pull request by hatianzhang**
# Description

Simple demo using Chroma MM vector store for MM retrieval

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [x] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

**Review by jerryjliu — APPROVED** [review]
this looks good! 

some nits: 
- the colab link is wrong (folder is `multi_modal`) 
- can we add a link to this notebook here: https://docs.llamaindex.ai/en/latest/use_cases/multimodal.html

File: docs/community/integrations/fleet_libraries_context.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+# Fleet Context Embeddings - Building a Hybrid Search Engine for the Llamaindex Library
```
Comment (jerryjliu) [inline docs/community/integrations/fleet_libraries_context.md:1]: you prob need to merge main

Comment (hatianzhang) [reply docs/community/integrations/fleet_libraries_context.md:1]: try to fix that pr's lint issue.
that pr breaks the main branch

Comment (hatianzhang) [reply docs/community/integrations/fleet_libraries_context.md:1]: @logan-markewich is this good

**Comment by hatianzhang** [comment]
> this looks good!
> 
> some nits:
> 
> * the colab link is wrong (folder is `multi_modal`)
> * can we add a link to this notebook here: https://docs.llamaindex.ai/en/latest/use_cases/multimodal.html

done!

**Pull request merged**